molecule/default/tests/test_dashboard.py

```diff
@@ -4,22 +4,43 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
 @pytest.mark.jira('asc-222')
-def test_cinder_service(host):
+def test_cinder_service_on_all(host):
```

Comment:
It is not necessary to rename this test method since the method itself cannot control the host set that is passed to it.